evaluator.go

```diff
@@ -1,227 +0,0 @@
-/*
-Copyright 2017 Google Inc. All rights reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-	http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package jsonnet
```

Comment:
I believe it ended up not being passed as a value (alloced on heap instead) because some functions weren't inlined (because they went via interfaces), but it has been a month now and I can't quite remember the details. Maybe it was possible to modify it to keep the abstraction without gc overhead but for only 2 params the value of it was quite low vs the cognitive overhead (and additional lines of code) of adding another abstraction.